lawlibrary/templates/peachjam/about.html

```diff
@@ -49,6 +49,17 @@ <h2 class="mb-4">{% trans 'JIFA' %}</h2>
       {% endblocktrans %}
     </p>
 
+    <h2 class="mb-4">{% trans 'Laws.Africa' %}</h2>
```

Comment:
Company names don't need to be translated.

```suggestion
    <h2 class="mb-4">Laws.Africa</h2>
```